miso-web/src/main/webapp/scripts/freezer_ajax.js

```diff
@@ -46,22 +47,23 @@
       save(!freezerJson || !freezerJson.id);
     });
   };
-  
-  Freezer.addRoom = function () {
-    Freezer.addRoomWithCallback(function() { Utils.showOkDialog('Room created', []); });
-  },
-  Freezer.addRoomWithCallback = function (callback) {
+
+  Freezer.addRoom = function() {
+    Freezer.addRoomWithCallback(function() {
+      Utils.showOkDialog('Room created', []);
+    });
+  }, Freezer.addRoomWithCallback = function(callback) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove use of this comma operator. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS878)